networks.json

```diff
@@ -2,7 +2,6 @@
   {
     "name": "mainnet1",
     "configFileUrl": "https://raw.githubusercontent.com/vegaprotocol/networks/master/mainnet1/mainnet1.toml",
-    "explorer": "https://explorer.vega.xyz/",
```

Comment:
Wht remove the link to the explorer here?

Reply:
IDK why this commit has appeared. It's completely not mine, as you can see there is no signature on this commit